web/src/main/resources/org/akaza/openclinica/i18n/words.properties

```diff
@@ -1593,6 +1593,8 @@ remove_filter = Remove Filter
 
 remove_previous_same_version_error = Remove Previous Same Version Error
 
+role_status = Role Status
```

Comment:
New strings should be defined at the bottom of the file to make it easier for people updating their translations to find them.